homeassistant/components/nut/const.py

```diff
@@ -453,6 +453,13 @@
         device_class=DEVICE_CLASS_TEMPERATURE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    "watts": SensorEntityDescription(
+        key="watts",
+        name="Watts",
+        native_unit_of_measurement=POWER_VOLT_AMPERE,
+        icon="mdi:flash",
```

Comment:
```suggestion
        device_class=DEVICE_CLASS_POWER
```

https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/__init__.py#L88